examples/src/bin/listview_applauncher.rs

```diff
@@ -0,0 +1,147 @@
+extern crate gtk;
+extern crate gdk;
+extern crate gio;
+
+use crate::gtk::prelude::*;
+use crate::gio::prelude::*;
+
+use std::env::args;
+
+pub fn main() {
+    gtk::init().unwrap_or_else(|_| panic!("Failed to initialize GTK."));
+
+    let provider = gtk::CssProvider::new();
+    provider.load_from_data("".as_bytes());
+
+    let display = gdk::Display::get_default().expect("Error getting default Display");
+    gtk::StyleContext::add_provider_for_display(&display, &provider,
+        gtk::STYLE_PROVIDER_PRIORITY_APPLICATION);
+
+
+    let application = gtk::Application::new(Some("org.gnome.Sdk"), Default::default())
```

Comment:
```suggestion
    let application = gtk::Application::new(Some("org.gtk-rs.examples.listview_applauncher"), Default::default())
```
or something like that